app/models/form_submission_attempt.rb

```diff
@@ -32,15 +32,22 @@ class FormSubmissionAttempt < ApplicationRecord
   end
 
   def log_status_change
-    Rails.logger.info(
-      {
-        name: 'Form Submissions Attempt State change',
-        form_submission_id:,
-        benefits_intake_uuid: form_submission&.benefits_intake_uuid,
-        from_state: aasm.from_state,
-        to_state: aasm.to_state,
-        event: aasm.current_event
-      }
-    )
+    log_hash = {
+      form_submission_id:,
+      benefits_intake_uuid: form_submission&.benefits_intake_uuid,
+      from_state: aasm.from_state,
+      to_state: aasm.to_state,
+      event: aasm.current_event
+    }
+    if aasm.to_state == 'failure'
+      log_hash[:name] = 'Form Submission Attempt failed'
+      Rails.logger.error(log_hash)
```

Comment:
My thinking here is that we'll most want to alert/monitor on `error`s rather than `success`es. This will let us use DD to target `@payload.name="Form Submission Attempt failed"`.

Reply:
Can you explain why we use `name` and not `message`?

I find `message makes it easier to read in Datadog

Reply:
I don't have a preference (don't know the difference) so I switched it 😃